src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -186,6 +195,7 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
               <CheckBoxCustom
                 value={isAllowFollowingSendDM}
                 label="Allow incognito only from users you follow?"
+                disabled={!isAllowFollowingSendDM && following < 20}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `disabled` prop is being used correctly here. The checkbox will be disabled if `isAllowFollowingSendDM` is `false` and `following` is less than 20.


<!-- This is an auto-generated comment by OSS CodeRabbit -->